lib/schema_plus/enums/middleware.rb

```diff
@@ -7,12 +7,12 @@ module Initial
         module Postgresql
 
           def after(env)
-            env.connection.enums.each do |schema, name, values|
+            env.connection.enums.sort_by {|k| k[1] }.each do |schema, name, values|
               params = [name.inspect]
               params << values.map(&:inspect).join(', ')
               params << ":schema => #{schema.inspect}" if schema != 'public'
 
-              env.initial << "create_enum #{params.join(', ')}"
+              env.initial << "  create_enum #{params.join(', ')}"
```

Comment:
Not sure if we would ever have more than one level of nesting on `Initial` and whether this should be evaluated more correctly?